app/controllers/evaluators_controller.rb

```diff
@@ -1,13 +1,14 @@
 # frozen_string_literal: true
 
-class ManageEvaluatorsController < ApplicationController
-  include ManageEvaluatorsHelper
-
+class EvaluatorsController < ApplicationController
   before_action -> { authorize_user('challenge_manager') }
-  before_action :set_phase, only: [:index, :create, :destroy]
+
+  # All routes are scoped to a Challenge Phase
+  before_action :set_challenge_phase
```

Comment:
relies on `phase_id` param from the nested route definition:
`/phases/:phase_id/evaluators/*`